code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -1,4 +1,4 @@
-var/list/hardcoded_gases = list("o2","n2","co2","plasma") //the main four gases, which were at one time hardcoded
+var/list/hardcoded_gases = list("o2","n2","co2","plasma", "freon") //the main four gases, which were at one time hardcoded
```

Comment:
thanks, but I was already a couple minutes into compiling with the list set to six. fml